ci-operator/config/jianzhangbjz/learn-operator/jianzhangbjz-learn-operator-master.yaml

```diff
@@ -0,0 +1,82 @@
+base_images:
+  base:
+    name: "4.7"
+    namespace: ocp
+    tag: base
+  cli:
+    name: "4.7"
+    namespace: ocp
+    tag: cli
+  operator-registry:
+    name: "4.7"
+    namespace: ocp
+    tag: operator-registry
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.14
+images:
+- dockerfile_path: build/Dockerfile.ci
+  from: base
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /tmp/learn-operator
+      - destination_dir: .
+        source_path: /tmp/bin
+  to: learn-operator
+- dockerfile_path: deploy/Dockerfile.src.ci
+  from: src
+  inputs:
+    cli:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/oc
+  to: learn-oc-bin-image
+operator:
+  bundles:
+  - context_dir: manifests/learn/0.0.1/
```

Comment:
> The operator.bundles stanza is a list, so it is possible to build multiple bundle images from one repository.

This sentence described in the [testing-operator-sdk-operators](https://docs.ci.openshift.org/docs/how-tos/testing-operator-sdk-operators/), so I thought `bundles` is a list and `context_dir` is for each bundle. But, get the errors below when setting `context_dir: manifests/learn/0.0.1/` 
```console
2020/10/26 11:49:22 Build learn-operator failed, printing logs:
2020/10/26 11:49:22 error: Unable to retrieve logs from failed build: build learn-operator is in an error state. No logs are available.
2020/10/26 11:49:32 No custom metadata found and prow metadata already exists. Not updating the metadata.
```
So, I guess this `context_dir` is for all bundles, not a specific bundle, right? @petr-muller

Reply:
Incorrect, `context_dir` is for a specific bundle and can be specifed multiple times, once per bundle. The errors you paste are generic "ci-operator" failed ones and do not say anything about a specific build, so I cannot investigate without being given a link to an actual failure.

But `Build learn-operator failed` means the **operator** build failed, not a bundle.